api/libproject/generate_appengine_cron_yaml.rb

```diff
@@ -9,12 +9,15 @@
 
 env = ARGV[0]
 
-cron_dir = "src/main/webapp/WEB-INF"
-to_cron_path = ->(suffix) { "#{cron_dir}/cron_#{suffix}.yaml" }
+CRON_DIR = "src/main/webapp/WEB-INF"
 
-path = to_cron_path.call("default")
-if File.file?(to_cron_path.call(env))
-  path = to_cron_path.call(env)
+def to_cron_path(suffix)
+  return "#{CRON_DIR}/cron_#{suffix}.yaml"
 end
 
-FileUtils.cp(path, "#{cron_dir}/cron.yaml")
+from_path = to_cron_path("default")
+if File.file?(to_cron_path(env))
```

Comment:
nit: This is a little confusing because of the reassignment and recalculation of the cron path. I might have defined the result of `to_cron_path(env)` and the default path separately, and then done the comparison and only assign `from_path` once. Lots of bigger fish to fry though.

Reply:
Totally. With a scope this small it's fine. I think I was thinking about something like `foo = a or b` but then remembered too late that `a` here won't be falsy when the file doesn't exist.